### ISSUE TITLE
kubectl delete -l crashes

### ISSUE THREAD

**Issue by bgrant0607** [GoogleCloudPlatform/kubernetes#3466]
```
$ cluster/kubectl.sh get pods
Running: cluster/../_output/local/bin/linux/amd64/kubectl get pods
POD                                     CONTAINER(S)        IMAGE(S)               HOST                LABELS                   STATUS
067403e7-9c1c-11e4-bb57-42010af0715c/   redis-slave         gurpartap/redis        127.0.0.1/          name=redis,role=slave    Unknown
0674cbf6-9c1c-11e4-bb57-42010af0715c/   redis-slave         gurpartap/redis        127.0.0.1/          name=redis,role=slave    Unknown
053c5954-9c1c-11e4-bb57-42010af0715c/   guestbook           kubernetes/guestbook   127.0.0.1/          name=guestbook           Unknown
053cd0d3-9c1c-11e4-bb57-42010af0715c/   guestbook           kubernetes/guestbook   127.0.0.1/          name=guestbook           Unknown
054085d6-9c1c-11e4-bb57-42010af0715c/   guestbook           kubernetes/guestbook   127.0.0.1/          name=guestbook           Unknown
redis-master-pod/                       redis-master        gurpartap/redis        127.0.0.1/          name=redis,role=master   Unknown
$ cluster/kubectl.sh delete pods -l=name=guestbook
Running: cluster/../_output/local/bin/linux/amd64/kubectl delete pods -l=name=guestbook
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x4cd65c]

goroutine 16 [running]:
runtime.panic(0x948c80, 0xcce753)
        /usr/local/go/src/pkg/runtime/panic.c:279 +0xf5
github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource.(*Builder).SelectorParam(0xc208044120, 0xc2080a6740, 0xf, 0xc2080a6740)
        kubernetes/_output/local/go/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource/builder.go:170 +0x27c
github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd.func·013(0xc2080ad7a0, 0xc208046200, 0x1, 0x2)
        /kubernetes/_output/local/go/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/delete.go:69 +0x279
github.com/spf13/cobra.(*Command).execute(0xc2080ad7a0, 0xc2080461a0, 0x2, 0x2, 0x0, 0x0)
        /kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:381 +0x2cb
github.com/spf13/cobra.(*Command).findAndExecute(0xc2080acfc0, 0xc20800e010, 0x3, 0x3, 0x0, 0x0)
        /kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:348 +0xd3
github.com/spf13/cobra.(*Command).Execute(0xc2080acfc0, 0x0, 0x0)
        /kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:434 +0x975
main.main()
        /kubernetes/_output/local/go/src/github.com/GoogleCloudPlatform/kubernetes/cmd/kubectl/kubectl.go:29 +0x76

goroutine 17 [runnable]:
runtime.MHeap_Scavenger()
        /usr/local/go/src/pkg/runtime/mheap.c:507
runtime.goexit()
        /usr/local/go/src/pkg/runtime/proc.c:1445

goroutine 18 [runnable]:
bgsweep()
        /usr/local/go/src/pkg/runtime/mgc0.c:1976
runtime.goexit()
        /usr/local/go/src/pkg/runtime/proc.c:1445

goroutine 19 [runnable]:
runfinq()
        /usr/local/go/src/pkg/runtime/mgc0.c:2606
runtime.goexit()
        /usr/local/go/src/pkg/runtime/proc.c:1445

goroutine 20 [runnable]:
github.com/golang/glog.(*loggingT).flushDaemon(0xcd5d40)
        /kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:878
created by github.com/golang/glog.init·1
        /kubernetes/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2b2

goroutine 17 [syscall]:
runtime.goexit()
        /usr/local/go/src/pkg/runtime/proc.c:1445

goroutine 21 [runnable]:
os/signal.loop()
        /usr/local/go/src/pkg/os/signal/signal_unix.go:19
created by os/signal.init·1
        /usr/local/go/src/pkg/os/signal/signal_unix.go:27 +0x32
```

/cc @smarterclayton

**Comment by bgrant0607** [comment]
I realize that I used the wrong syntax, but it's an easy mistake. Basically, the label selector was empty.

**Comment by smarterclayton** [comment]
Same as #3460

----- Original Message -----

**Comment by lavalamp** [comment]
Should be fixed now, #3460 merged
